src/zhinst/toolkit/driver/nodes/generator.py

```diff
@@ -94,14 +94,22 @@ def load_sequencer_program(
         """Compiles and loads a sequencer program.
 
         Args:
-            sequencer_program: Sequencer program to be uploaded
+            sequencer_program: Sequencer program to be uploaded.
             timeout: Maximum time to wait for the compilation on the device in
-                seconds. (default = 10s)
+                seconds.
 
         Raises:
+            ValueError: `sequencer_program` is an empty string.
             TimeoutError: If the upload or compilation times out.
             RuntimeError: If the upload or compilation failed.
+
+        .. versionadded:: 0.3.4
+
+            `sequencer_program` does not accept empty strings
+
         """
+        if sequencer_program == "":
```

Comment:
```suggestion
        if not sequencer_program:
```
What speaks against that?

Reply:
That is a good suggestion.